app/assets/stylesheets/kudos/components/_kudo.scss

```diff
@@ -46,6 +46,7 @@
   border-radius: .25rem;
   box-shadow: 0 2px 4px 0 rgba(0, 0, 0, .1);
   width: 100%;
+  min-width: 0;
```

Comment:
Seemed to be the "trick" to prevent super long words from making the Kudos flex box larger width than intended

https://css-tricks.com/flexbox-truncated-text/